### DISCUSSION TITLE
TTS-55 Pro  Laser runs but does not engrave

### DISCUSSION THREAD

**Discussion by 2Together** [arkypita/LaserGRBL#2265] [General]
I recencly bought a TTS-55 Pro i run it with laserGRBL and the laser runs but does no engrave, i leave a video, please someone could help me , thanks
https://github.com/arkypita/LaserGRBL/assets/37485049/3186d637-a3df-447f-b194-9c605e6bc5e7

**Comment by rotkop456** [comment]
Hi @2Together , I also have a TTS-55 Pro and my engraving is working 100%. This is the setting I am using hope this helps you. The higher the S-Max the darker and deeper its going to burn, the lower the S-Max the more shallow it will engrave.

![1- Import Raster Image](https://github.com/arkypita/LaserGRBL/assets/9168743/4e4a0a10-45a3-4f19-86a9-abadcabfdbd1)
![2- Target image](https://github.com/arkypita/LaserGRBL/assets/9168743/e26cfe41-8a67-41de-a586-bdb05bd88003)
